langlib/langlib-test/src/test/java/org/ballerinalang/langlib/test/LangLibValueTest.java

```diff
@@ -325,42 +325,43 @@ public void testCloneWithType(String function) {
 
     @DataProvider(name = "cloneWithTypeFunctions")
     public Object[][] cloneWithTypeFunctions() {
-        return new Object[][] {
-                { "testCloneWithTypeJsonRec1" },
-                { "testCloneWithTypeJsonRec2" },
-                { "testCloneWithTypeOptionalFieldToMandotoryField" },
-                { "testCloneWithTypeAmbiguousTargetType" },
-                { "testCloneWithTypeForNilPositive" },
-                { "testCloneWithTypeForNilNegative" },
-                { "testCloneWithTypeNumeric1" },
-                { "testCloneWithTypeNumeric2" },
-                { "testCloneWithTypeNumeric3" },
-                { "testCloneWithTypeNumeric4" },
-                { "testCloneWithTypeNumeric5" },
-                { "testCloneWithTypeNumeric6" },
-                { "testCloneWithTypeNumeric7" },
-                { "testCloneWithTypeToArrayOfRecord" },
-                { "testCloneWithTypeToArrayOfMap" },
-                { "testCloneWithTypeIntArrayToUnionArray" },
-                { "testCloneWithTypeIntSubTypeArray" },
-                { "testCloneWithTypeStringArray" },
-                { "testCloneWithTypeWithInferredArgument" },
-                { "testCloneWithTypeWithImmutableTypes" },
-                { "testCloneWithTypeDecimalToInt"},
-                { "testCloneWithTypeDecimalToIntNegative" },
-                { "testCloneWithTypeDecimalToByte"},
-                { "testCloneWithTypeDecimalToIntSubType"},
-                { "testCloneWithTypeTupleToJSON"},
-                { "testCloneWithTypeImmutableStructuredTypes"},
-                { "testCloneWithTypeWithFiniteArrayTypeFromIntArray" },
-                { "testCloneWithTypeWithFiniteType" },
-                { "testCloneWithTypeWithUnionOfFiniteType" },
-                { "testCloneWithTypeWithFiniteArrayTypeFromIntArray" },
-                { "testCloneWithTypeWithUnionOfFiniteTypeArraysFromIntArray" },
-                { "testCloneWithTypeWithUnionTypeArrayFromIntArray" },
-                { "testCloneWithTypeWithFiniteTypeArrayFromIntArrayNegative" },
-                { "testConvertJsonToNestedRecordsWithErrors" },
-                { "testCloneWithTypeNestedStructuredTypesNegative"}
+        return new Object[][]{
```

Comment:
Can we return `String[]` instead of `Object[][]` since we are passing strings as function names?